tests/setup_cfg_fmt_test.py

```diff
@@ -227,6 +228,9 @@ def test_normalize_lib(lib, expected):
     ),
 )
 def test_adds_long_description_with_readme(filename, content_type, tmpdir):
+    dir_name = os.path.splitext(os.path.basename(filename))[0]
```

Comment:
for some reason this case wasn't covered by the test suite, so the tox environment invocation failed because ``100%`` coverage is enforced on the project.